CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 cmake_minimum_required (VERSION 3.20 FATAL_ERROR)
 project(AMR-Wind CXX C)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON CACHE INTERNAL "")
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
```

Comment:
Are you fine with this (and other cmake changes) @jrood-nrel ?

Reply:
Maybe I'm confused. Is ERF using `add_subdirectory` on AMR-Wind?

Reply:
Actually I am fine with `PROJECT_SOURCE_DIR`. It's probably the right move in all our projects.

Reply:
The ERF/AMR-Wind coupling driver is using find_package on AMR-Wind and linking to the library dynamically (for now). See https://github.com/erf-model/erf-amrwind-driver/blob/384327aeddb1ad22a2aa54b71cd62b3acc049dd0/CMake/BuildExe.cmake#L205.